css/css-grid/alignment/grid-baseline-align-001.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<title>
+	Grid Item (First) Baseline Block-Axis Alignment: auto-height grid, auto row, auto items
+</title>
+<link rel="help" href="https://drafts.csswg.org/css-grid-1/#column-align">
+<link rel="help" href="https://www.w3.org/TR/css-grid-1/#algo-content">
+<link rel="help" href="https://www.w3.org/TR/css-align-3/#baseline-values">
+<link rel="match" href="references/grid-baseline-align-001-ref.html">
```

Comment:
I believe we miss an assert text to explain what this is specifically testing (despite with the PR title it's clear).